style/modules/muralModal.css

```diff
@@ -0,0 +1,69 @@
+.muralModal {
+    display: flex;
+    max-width: 1250px;
+    max-height: 1000px;
+    color: #606060;
+    position: relative;
+
+}
+
+.muralModal--name {
+    font-size: 28px;
+    font-weight: bold;
+    color: #113255;
+}
+
+.muralModal--container {
+    padding: 35px 35px;
+    background: white;
+}
+
+.muralModal--end {
+    padding-top: 105px;
+}
+
+.muralModal--bold {
```

Comment:
not sure about the SMACCS rules but would bold maybe be a variation of text? or is bold somehow different? what were you thinking here @kingtrentiii

Reply:
I mentioned this on Tuesday but should add here that I'm personally fine with it.

Tradeoff is:
- if it's a variant, you would have to style the HTML with e.g. `<div class="muralModal--text muralModal--text-bold">`, which is a little clunky
- if it's not, the `font-size: 20px;` rule is duplicated between classes, so there's a risk during updates of one of those getting missed

I think in either case the downside is not that risky, so IMO it's down to taste.

Reply:
Side note bc I think it's interesting: the risk of missing updates is just a general problem with cloned code. In academia they call it [late propagation](https://ieeexplore.ieee.org/document/6080794) when the forgotten instance eventually gets updated. Managing clones is an interesting research problem, and I wish the detection tools were in a little wider use.

Then again, in my experience, you know *where* the duplication is, it's just that you don't always have time to fix it.

Anyway... another way this sort of thing (the bolding, I mean) gets done is with utility classes, which I'm not a huge fan of. Here's an example of making something bold in Boostrap: https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics

By using SMACSS, we're intentionally taking the opposite approach -- breaking down the pages into large-ish, high-level components.

*Another* way this can be done is with e.g. [Sass mixins](https://sass-lang.com/documentation/at-rules/mixin), which are like CSS variables on steroids. (Sass is great, but using it would have added a build step to the project.)